sunpy/map/sources/goes.py

```diff
@@ -0,0 +1,92 @@
+"""GOES SXI Map subclass definitions"""
+from __future__ import absolute_import, division, absolute_import
+#pylint: disable=W0221,W0222,E1101,E1121
+
+__author__ = "[Iain Hannah, Steven Christe]"
+__email__ = "steven.christe@nasa.gov"
+
+from matplotlib.colors import LogNorm
+
+import astropy.units as u
+
+from sunpy.map import GenericMap
+from sunpy.cm import cm
+from sunpy.coordinates import get_sunearth_distance, get_sun_B0
+from sunpy import sun
+
+__all__ = ['SXIMap']
+
+
+class SXIMap(GenericMap):
+    """GOES SXI Image Map
+
+    The GOES Solar X-ray Imager (SXI) provides uninterrupted, full-disk, soft X-ray
+    solar images, with a 1 min cadence and a single-image (adjustable) dynamic
+    range near 100. A set of metallic thin-film filters provides temperature
+    discrimination in the 0.6–6.0 nm bandpass. The spatial resolution of
+    approximately 10 arcsec Full width half maximum (FWHM) is sampled with 5
+    arcsec pixels.
+
+    SXI was first launched on 23 July 2001 on NOAA’s GOES-12 satellite and has
+    obtained uninterrupted images starting on 22 January 2003.
+
+    References
+    ----------
+    * `Mission Instrument Page <https://sxi.ngdc.noaa.gov/index.html>`_
+    * `Data notes and Fits headers <https://sxi.ngdc.noaa.gov/sxi_data_notes.html>`_
+    * `Official SXI Data Book <https://sxi.ngdc.noaa.gov/images/section06.pdf>`_
+    * `Instrument Paper DOI: 10.1007/s11207-005-7416-x <https://link.springer.com/article/10.1007/s11207-005-7416-x>`_
+    * `SXI Performance Paper DOI: 10.1007/s11207-005-7417-9 <https://link.springer.com/article/10.1007/s11207-005-7417-9>`_
```

Comment:
Instead of 'springer.com' I was thinking of: `http://dx.doi.org/<doi>`

Reply:
Thanks! I was not aware of that service.